modules/setting/setting.go

```diff
@@ -262,8 +262,8 @@ var (
 	}{
 		ExplorePagingNum:    20,
 		SitemapPagingNum:    20,
-		IssuePagingNum:      10,
-		RepoSearchPagingNum: 10,
+		IssuePagingNum:      20,
+		RepoSearchPagingNum: 20,
```

Comment:
No corresponding config option?

Reply:
yeah, this `RepoSearchPagingNum` config seems to be undocumented for years now 🙃